### PR TITLE
Replace Google TM code with Matomo TM code

### PR DESCRIPTION
**Pull request by katleonard**
Replacing Google Tag Manage code with Matomo Tag Manager code

### CODE REVIEW

**Comment by HaSistrunk** [comment]
Ok! I see a few things to tweak:

1. I'm realizing that we don't need the "if statement" in lines 2 and 12 at all (the `if jekyll.environment == "production"` stuff). This is a conditional statement that prevents the Matomo script from running unless the site is deployed on the live production environment. However, in our setup we are actually controlling that in the Matomo settings for the website instead of here in the site code. So you can delete those two lines!

2. The Matomo script you used doesn't look like it's the one that corresponds to the Library Site container in Matomo. Each site in Matomo Tag Manager has a unique container id, so that script in the "Install Code" menu in Matomo is different depending on which website container you have selected in Tag Manager.

3. There is a Google Tag Manager (noscript) block of code that also needs to be deleted in a different file (`_layouts/default.html`) lines 6-9. We don't need to replace that with anything - it's an extra bit of code that Google requires and Matomo does not.

File: _includes/head.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-    })(window,document,'script','dataLayer','GTM-NQRWDJK');</script>
-    <!-- End Google Tag Manager -->
-  {%- endif -%}
-
+ <!-- Matomo Tag Manager -->
+<script>
+var _mtm = window._mtm = window._mtm || [];
+_mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+g.async=true; g.src='https://cdn.matomo.cloud/rockarch.matomo.cloud/container_SdzKujX7_dev_cc9545323ac9afaa195e6699.js'; s.parentNode.insertBefore(g,s);
```
Comment (HaSistrunk) [inline _includes/head.html]:
```suggestion
g.async=true; g.src='https://cdn.matomo.cloud/rockarch.matomo.cloud/container_SdzKujX7.js'; s.parentNode.insertBefore(g,s);
```

Comment (HaSistrunk) [reply _includes/head.html]: We can just use the live container environment. Happy to talk through our approach in the next sprint meeting.

**Review by daremiyo — APPROVED** [review]
Looks good with the changes you've made!

**Review by HaSistrunk — APPROVED** [review]

**Review by EcZX — APPROVED** [review]

**Pull request merged**